src/web/app/components/question-types/question-edit-details-form/mcq-question-edit-details-form.component.ts

```diff
@@ -0,0 +1,132 @@
+import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
+import { Component, OnInit } from '@angular/core';
+import {
+  FeedbackMcqQuestionDetails,
+  FeedbackParticipantType,
+} from '../../../../types/api-output';
+import { DEFAULT_MCQ_QUESTION_DETAILS } from '../../../../types/default-question-structs';
+import { QuestionEditDetailsFormComponent } from './question-edit-details-form.component';
+
+/**
+ * Question details edit form component for Mcq question.
+ */
+@Component({
+  selector: 'tm-mcq-question-edit-details-form',
+  templateUrl: './mcq-question-edit-details-form.component.html',
+  styleUrls: ['./mcq-question-edit-details-form.component.scss'],
+})
+export class McqQuestionEditDetailsFormComponent
+    extends QuestionEditDetailsFormComponent<FeedbackMcqQuestionDetails> implements OnInit {
+
+  readonly PARTICIPANT_TYPES: string[] = [FeedbackParticipantType.STUDENTS,
+    FeedbackParticipantType.STUDENTS_EXCLUDING_SELF, FeedbackParticipantType.TEAMS,
+    FeedbackParticipantType.TEAMS_EXCLUDING_SELF, FeedbackParticipantType.INSTRUCTORS];
+
+  constructor() {
+    super(DEFAULT_MCQ_QUESTION_DETAILS());
+  }
+
+  ngOnInit(): void {
+    if (this.model.numOfMcqChoices === 0) {
```

Comment:
Not sure if you should add this here... this is adding an unexpected logic for the component.

Reply:
Can I change the Default structure for Mcq question in that case? The idea is to have at least 2 Mcqs displayed to the user when the question is first selected

Reply:
Why do you need to change the default structure? If you need at least 2 MCQs displayed, then modify the model when you're passing it to this component.